.all-contributorsrc

```diff
@@ -18,6 +18,15 @@
         "infra",
         "test"
       ]
+    },
+    {
+      "login": "gyandeeps",
```

Comment:
Is it ok if I add you as a contributor @gyandeeps? See [this](https://github.com/kentcdodds/prettier-eslint/blob/32c2002de7eab58047226e80b55ade257ebfdcb1/README.md#contributors) for what it'll look like.